samples/stock/interactive-docs/range-selector/demo.html

```diff
@@ -0,0 +1,9 @@
+<script src="https://code.highcharts.com/stock/highstock.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+
+<div id="container" class="chart"></div>
+<div id="button-box">
+    <button id="toggle-range-selector">Disable range selector</button>
```

Comment:
Is it my Chrome, or buttons are missing cursor: pointer + hover state?

Edit: In all demos